### PR TITLE
SKALE-4124 Remove gas limit and gas price to set domain cmd

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: node_cli/cli/node.py [hunk 1]
```diff
@@ -229,12 +229,24 @@ def _turn_on(maintenance_off, sync_schains, env_file):
     type=str,
     help='Node domain name'
 )
+@click.option(
+    '--gas-limit',
+    default=None,
+    type=int,
+    help='Gas limit for registration transaction'
```
Comment (badrogger) [inline node_cli/cli/node.py]: typo: "registration transaction" -> "set domain transaction"

**Review by badrogger — CHANGES_REQUESTED** [review]

File: node_cli/cli/node.py (hunk 1 above)
Comment (dmytrotkk) [reply node_cli/cli/node.py]: fixed, thanks!

**Review by badrogger — APPROVED** [review]

File: text.yml [hunk 2]
```diff
@@ -60,3 +60,10 @@ exit:
     in_progress: "Node exiting is in progress"
     wait_for_rotations: "Node is waiting to finish rotations"
     completed: "Node exiting is completed"
+
+gas_price:
```
Comment (badrogger) [inline text.yml]: Looks like this lines can be removed to

**Review by badrogger — CHANGES_REQUESTED** [review]

File: text.yml (hunk 2 above)
Comment (dmytrotkk) [reply text.yml]: done

**Review by badrogger — APPROVED** [review]

**Pull request merged**